InWorldz/InWorldz.Phlox.Engine/LSLSystemAPI.cs

```diff
@@ -15494,24 +15493,24 @@ public string iwStringCodec(string str, string codec, int operation, LSL_List ex
                             }
                         }
                     }
-                    if (aesKey == "" || aesVector == "" || aesCodec == "")
+                    if (String.IsNullOrEmpty(aesKey) || String.IsNullOrEmpty(aesVector) || String.IsNullOrEmpty(aesCodec))
```

Comment:
No semantic change assuming LSL cannot send null strings through an LSL_List.  If it can, this this improves the check as that means the needed params are missing/blank.